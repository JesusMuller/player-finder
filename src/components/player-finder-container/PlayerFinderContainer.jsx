import React from 'react';
import SectionTitle from '../section-title';
import SearchFormContainer from '../player-search-form';
import PlayerInformationTable from '../player-information-table';
import './PlayerFinderContainer.scss';

const PlayerFinderContainer = () => (
    <div className="section-container">
        <SectionTitle textToShow="Football Player Finder"/>
        <SearchFormContainer />
        <PlayerInformationTable />
    </div>
);

export default PlayerFinderContainer;
