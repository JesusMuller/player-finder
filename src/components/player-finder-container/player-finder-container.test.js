import React from 'react';
import { shallow } from 'enzyme';
import PlayerFinderContainer from './player-finder-container';

describe('<PlayerFinderContainer />', () => {
    it('Should match snapshot', () => {
        const wrapper = shallow(<PlayerFinderContainer />);
        expect(wrapper).toMatchSnapshot();
    });
});
