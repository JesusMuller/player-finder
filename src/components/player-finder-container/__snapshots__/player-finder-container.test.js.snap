// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlayerFinderContainer /> Should match snapshot 1`] = `
<div
  className="section-container"
>
  <SectionTitle
    textToShow="Football Player Finder"
  />
  <ConnectFunction />
  <ConnectFunction />
</div>
`;
